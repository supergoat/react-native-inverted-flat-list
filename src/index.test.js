// @flow
import React from 'react';
import InvertedFlatList from './index';
import renderer from 'react-test-renderer';

describe('InvertedFlatList', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<InvertedFlatList />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
