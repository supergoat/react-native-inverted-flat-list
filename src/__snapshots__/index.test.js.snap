// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InvertedFlatList renders correctly 1`] = `
<View>
  <Text>
    Hello World!
  </Text>
</View>
`;
